src/components/project/project.service.ts

```diff
@@ -537,10 +545,13 @@ export class ProjectService {
         q
           .match([
             node('node'),
-            relation('out', '', sort),
+            relation('out', '', sort, { active: true }),
```

Comment:
Is this necessary? If so it should be applied to every list

Reply:
yes. When we traverse over property relationships we must always test for active true in order to return only non-deleted properties.